client/data/account-keys/actions.js

```diff
@@ -55,7 +55,7 @@ export function* saveAccountKeys( accountKeys ) {
 			data: accountKeys,
 		} );
 
-		if ( ! accountData?.id ) {
+		if ( ! isDisconnecting && ! accountData?.id ) {
```

Comment:
I noticed this error when testing this PR so snuck a fix into this PR.

To replicate the issue this change is fixing:

1. On `develop` go to **WooCommerce > settings > Payments > Stripe > Settings**
2. Open the Account keys modal. 
3. Delete your keys. 
4. Save.
5. On develop you will get this error:
<p align="center">
<img width="332" alt="Screenshot 2024-06-25 at 5 07 48 PM" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/8490476/c2bb7259-59bd-4233-afe9-08bf0c8cbe8c">
</p>

It turns out this error was caused by this `throw 'Account not Found';`. When disconnecting the account, no account data being returned is the expected outcome and so no error should be thrown in that case.

Reply:
Niiiice. Good thing to sneak in. Working great, thankya!